upb/handlers.h

```diff
@@ -633,6 +634,10 @@ UPB_BEGIN_EXTERN_C
 /* Native C API. */
 
 /* Handler function typedefs. */
+typedef bool upb_addunknown_handlerfunc(void *c, const void *hd,
+                                        const char *buf, size_t n);
+typedef bool upb_unknown_handlerfunc(void *c, const void *hd, const char *buf,
```

Comment:
What is the difference between these two? They are the same typedef, do they have any difference in meaning?

Reply:
Removed